core/pubnub_ccore_pubsub.c

```diff
@@ -122,8 +122,8 @@ char const* pbcc_get_msg(struct pbcc_context* pb)
             if (NULL != pb->crypto_module) {
                 char* trimmed = (char*)malloc(strlen(rslt) + 1); // same length as rslt
                 if (NULL == trimmed) {
-                    PUBNUB_LOG_ERROR("pbcc_get_msg(pbcc=%p) - failed to allocate memory for trimmed string. Dropping message!\n", pb);
-                    return NULL;
```

Comment:
I'm not really sure about this one because the reason of error is not issues with source message. We need to report issues with decryption. Allocation or buffer size should be treated the same.

Reply:
Okay, so Should I return just error with NULL, yep?